src/components/scenegraph/Toolbar.js

```diff
@@ -69,13 +82,42 @@ export default class Toolbar extends React.Component {
    * Try to write changes with aframe-inspector-watcher.
    */
   writeChanges = () => {
-    const xhr = new XMLHttpRequest();
-    xhr.open('POST', 'http://localhost:51234/save');
-    xhr.onerror = () => {
-      alert('aframe-watcher not running. This feature requires a companion service running locally. npm install aframe-watcher to save changes back to file. Read more at supermedium.com/aframe-watcher');
-    };
-    xhr.setRequestHeader('Content-Type', 'application/json');
-    xhr.send(JSON.stringify(AFRAME.INSPECTOR.history.updates));
+    const baseUrl = "http://localhost:8888/";
+    const apiEndpointScene = AFRAME.scenes[0].getAttribute("id").replace("-scene", "");
+    const apiEndpoint = "api/admin/v1/scene/";
+    const getUrl = baseUrl + apiEndpoint + apiEndpointScene;
+    let objects = [];
+    let objectChanges = [];
+    axios.get(getUrl, {
```

Comment:
This is going to make a call to do a GET on each save right? Like the workflow here is:
1. Admin hits save button
2. We do a GET on the scene to get all the objects
3. We iterate through the changes and update the corresponding objects

That would be a GET on each save. Instead can we do something like:
1. Do an initial GET on the scene to get all the objects and save in state
2. Iterate through the changes and update the state, make a PUT request for any objects that are changed

That way we don't have to make a GET request on each save?